python/cudf/io/csv.py

```diff
@@ -460,6 +471,7 @@ def read_csv_strings(filepath_or_buffer, lineterminator='\n',
     csv_reader.skipfooter = skipfooter
     csv_reader.decimal = decimal.encode()
     csv_reader.thousands = thousands.encode() if thousands else b'\0'
+    csv_reader.nrows = nrows if nrows else -1
```

Comment:
`nrows` can technically be 0 as a valid value to return an empty dataframe with column names and dtypes. We don't want to trigger the -1 clause in that case.

Reply:
I don't think that usage would work with the current implementation. I'll look into this.

Reply:
Fixed in latest version.